tests/System.Buffers.Primitives.Tests/MemoryTests.cs

```diff
@@ -167,59 +167,6 @@ private static IEnumerable<object[]> MakePermutations(params object[][] argss)
         }
         #endregion
 
-        public static IEnumerable<object[]> ReservationPerformanceData =>
```

Comment:
One thing I forgot to explain: the tests don't work as it because AutoPooledMemory does not use ReferenceCounter